exampleSite/content/fragments/stripe/docs.md

```diff
@@ -8,6 +8,92 @@ title = "Documentation"
   enable = true
 +++
 
-# UNDER CONSTRUCTION
+Stripe fragment creates a form with a Stripe element inside it and is used to create payments for the users.
+
+It requires a backend for creating the customer and charge for Stripe. An example can be found at [here](https://github.com/okkur/syna-stripe-gcf) which can be deployed on Google Cloud Functions and works out of the box.
+
+The use of a backend is also useful for saving other information from the user such as the product they selected (in case you are providing more than one products) or other metadata.
+
+### Events
+
+This fragment uses the [Events](/docs/events) api by default.
+
+#### Subscribers
+
+##### pricing:change
+
+This event is published by the [Pricing fragment](/fragments/pricing/#events). It can also be published by [event urls](/docs/events/#event-urls).
+
+###### product
+*type: string*
+
+If provided, this variable will override the preconfigured `product` variable and will be displayed on top of the form.
+
+###### description
+*type: string*
+
+Payment metadata. This varialbe is not displayed to the user but will be sent to the server with the form data.
+
+###### price
+*type: string*
+
+The price that will be charged to the user's account.
+
+###### currency
+*type: string*
+
+The currency in which the user will be charged.
```

Comment:
Maybe add a small note how the price value and it's corresponding currency such as "$" is only user facing and the currency variable is essential for determining the charge. What do you think?